README.md

```diff
@@ -2,22 +2,18 @@
 
 This repo contains the Contiv website source files and build scripts. If you want to make changes to the Contiv public website, make changes to the source files here and follow the instructions below.
 
-Note: The following intstructions were tested on Mac OSX. If you are using a Windows environment, you will have to make adjustments.
-
 ## Prerequisites
 
-- Ruby 2.2.0 or higher. Ruby comes preinstalled with OSX. If you have Xcode installed on your Mac, you can install Ruby using "xcode-select --install" command if it isn't there already.
-- Bundler 1.11 or higher ("sudo gem install bundler")
-- Middleman 3.4 or higher ("sudo gem install middleman")
-
+- Ruby 2.0.0.  You must use exactly `2.0.0`.  You may want to install it via `rvm` or `rbenv`.
+- Bundler (`gem install bundler` after setting up Ruby 2.0.0)
+- Install the rest of the dependencies: `cd websrc; bundle`
 
 ## Running the Site Locally
 
 Running the site locally is simple. Clone this repo and run `cd websrc; make dev`.
 
 Then open up `http://localhost:4567`.
 
-
 ## Submitting Changes
 
 Make the changes locally on your machine and test it using the above instructions.
```

Comment:
Make the changes locally to the .md files on your machine and test it using the the above instructions.